lib/pbench/server/api/resources/__init__.py

```diff
@@ -801,6 +803,8 @@ def _dispatch(self, method: Callable, request: Request) -> Response:
             abort(HTTPStatus.BAD_REQUEST, message="Invalid request payload")
 
         try:
+            if json_data:
+                json_data.update(uri_encoded)
```

Comment:
Yeah I think It should be 
```
            if json_data:
                json_data.update(uri_encoded)
            else:
                json_data = uri_encoded
```

Reply:
As long as `json_data` is never `None` and is `{}` when it is empty/unspecified, then
```
            json_data.update(uri_encoded)
```
unconditionally would be nice and simple.

If it can be `None`, then what you suggest would be better.